src/Morphir/Visual/ViewApply.elm

```diff
@@ -70,7 +80,7 @@ view config viewValue functionValue argValues =
                                     el ((Font.color <| rgb 0.8 0 0) :: popupstyles) (text <| Error.toString err)
 
                         _ ->
-                            el popupstyles (text <| "Could not evaluate this function. (" ++ Error.toString firstError ++ ")")
+                            el popupstyles (text <| "Couldn't evaluate this function. (" ++ Error.toString firstError ++ ")")
```

Comment:
Very minor comment but in formal text it is preferred not to use the contracted form so "could not" was intentional here.